language-support/java/bindings-rxjava/BUILD.bazel

```diff
@@ -34,6 +34,7 @@ da_java_library(
     deps = [
         "//language-support/java/bindings:bindings-java",
         "//ledger-api/rs-grpc-bridge",
+        "//ledger/ledger-api-auth-client",
```

Comment:
One of the principles for the java bindings was to not import any scala libraries because it could lead to conflicts of transitive dependencies for users (e.g. scala-library 2.12 vs 2.13).

Reply:
Makes sense, I'll turn it into a Java library.

Reply:
Addressed by https://github.com/digital-asset/daml/commit/425302cd96a4b2ef047eaa52fd9037668e43feb8